lib/tests/streamlit/elements/chat_test.py

```diff
@@ -12,17 +12,28 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""checkbox unit tests."""
+"""chat input and message unit tests."""
 
 from parameterized import parameterized
 
 import streamlit as st
 from streamlit.errors import StreamlitAPIException
 from streamlit.proto.Block_pb2 import Block as BlockProto
 from tests.delta_generator_test_case import DeltaGeneratorTestCase
+import pytest
```

Comment:
Some of the import are duplicated here.

Reply:
Ah. I wanted to let precommit do all the work. Done :-)